setup.py

```diff
@@ -3,10 +3,29 @@
 from Cython.Build import cythonize
 
 import versioneer
+import sys
+
+
+def find_numba():
+    numba_dir = ""
+    sys_packages = sys.path
+    for pcg in sys_packages:
+        if pcg.find("/numba-0") != -1:
+            numba_dir = pcg
+    return numba_dir
```

Comment:
@1e-to please use `numba.core.extending.include_path()`

Reply:
Docs about it - https://numba.pydata.org/numba-doc/dev/developer/numba-runtime.html?highlight=include_path#using-the-nrt-from-c-code